### PR TITLE
Proper check for gem installation

### PR DESCRIPTION
**Pull request by ivanssimple**
When /etc/chef exists, chef-formatters-simple is not installed. This change will prevent that from happening.

I also updated mkdir to include the permissions so chmod is no longer needed.

### CODE REVIEW

File: littlechef/solo.py [hunk 1]
```diff
@@ -74,7 +74,9 @@ def configure(current_node=None):
     if not exists(logging_path):
         sudo('mkdir -p {0}'.format(logging_path))
     if not exists('/etc/chef'):
-        sudo('mkdir -p /etc/chef && chmod -R 774 /etc/chef && /opt/chef/embedded/bin/gem install chef-formatters-simple')
+        sudo('mkdir -m 774 -p /etc/chef')
+    if not 'true' in subprocess.check_output("/opt/chef/embedded/bin/gem list -eqi chef-formatters-simple; exit 0",stderr=None, shell=True):
```
Comment (ptqa) [inline littlechef/solo.py]: won't this always be true, because of `exit 0`?

Comment (ulmen) [reply littlechef/solo.py]: no, he is not checking the exit code here, he is checking actual output of `gem list -eqi chef-formatters-simple`, which returns "true" if gem is installed. but why `exit(0)` is needed in this case, @ivanssimple ?

Comment (ptqa) [reply littlechef/solo.py]: Gotcha. exit 0 is probably just not to get exception. I think we can just check exit code here and not parse output:
```
if 1 == subprocess.call("/opt/chef/embedded/bin/gem list -eqi chef-formatters-simple",stderr=None, shell=True)
```

Comment (ivanssimple) [reply littlechef/solo.py]: Updated.

**Pull request merged**